cli/src/cli.ts

```diff
@@ -83,38 +75,13 @@ async function runWebClient(args: Args): Promise<void> {
   let serverOk: unknown;
 
   try {
-    server = spawnPromise(
-      "node",
-      [
-        inspectorServerPath,
-        ...(args.command ? [`--env`, args.command] : []),
-        ...(args.args ? [`--args=${args.args.join(" ")}`] : []),
-      ],
-      {
-        env: {
-          ...process.env,
-          PORT: SERVER_PORT,
-          MCP_ENV_VARS: JSON.stringify(args.envArgs),
-        },
-        signal: abort.signal,
-        echoOutput: true,
-      },
-    );
-
-    // Make sure server started before starting client
-    serverOk = await Promise.race([server, delay(2 * 1000)]);
-  } catch (error) {}
-
-  if (serverOk) {
-    try {
-      await spawnPromise("node", [inspectorClientPath], {
-        env: { ...process.env, PORT: CLIENT_PORT },
-        signal: abort.signal,
-        echoOutput: true,
-      });
-    } catch (e) {
-      if (!cancelled || process.env.DEBUG) throw e;
-    }
+    await spawnPromise("node", [inspectorClientPath], {
+      env: { ...process.env, PORT: CLIENT_PORT },
```

Comment:
I think we don't need the `PORT: CLIENT_PORT` because we already read those vars from `process.env` in `start.js`?

https://github.com/modelcontextprotocol/inspector/blob/1aea3374ff14b3bd0f50d2236cf9c44d5a362c0f/client/bin/start.js#L241